src/CommunityToolkit.Maui.SourceGenerators/Helpers/SourceGeneratorHelpers.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+
+namespace CommunityToolkit.Maui.SourceGenerators.Helpers
+{
+	static class SourceGeneratorHelpers
```

Comment:
Let's avoid creating a file called SourceGeneratorHelpers because it'll just become a dumping ground for future PRs.

Reply:
Sure, I'll create a ParseHelper (?) or something like that to handle the generated `string`